packages/coinstac-ui/app/render/state/ducks/util.js

```diff
@@ -1,7 +1,11 @@
 import app from 'ampersand-app';
 
+export const logUI = (level, msg) => {
+  app.renderLogger[level](msg);
```

Comment:
Why expose  the logger as a two-argument function as opposed the default level-methods? Calls look cleaner:

```js
import { logUI as log } from '../state/ducks/util';
// ...
log('Imma message');
log.error('Oh noes!');
```

…versus:

```js
import { logUI as log from '../state/ducks/util';
// ...
log('info', 'Imma message');
log('error', 'Oh noes!');
```

This also means that one must look for two cases when attempting to track down log calls (`error`, for example: `log\.error` and `log\('error'`).